airbyte-integrations/connectors/destination-snowflake/src/main/kotlin/io/airbyte/integrations/destination/snowflake/typing_deduping/SnowflakeSqlGenerator.kt

```diff
@@ -82,48 +79,28 @@ class SnowflakeSqlGenerator(private val retentionPeriodDays: Int) : SqlGenerator
     }
 
     override fun createSchema(schema: String): Sql {
-        return of(
-            StringSubstitutor(java.util.Map.of("schema", StringUtils.wrap(schema, QUOTE)))
-                .replace("CREATE SCHEMA IF NOT EXISTS \${schema};")
-        )
+        return of("CREATE SCHEMA IF NOT EXISTS \"$schema\";")
```

Comment:
nit: triplequoted strings could avoid the escape
```suggestion
        return of("""CREATE SCHEMA IF NOT EXISTS "$schema";""")
```